avalanche's k8s deployment yaml file

ravishankarsrrav opened the pull request.

csmarchbanks commented on kubernetes-deployment-yaml:
```diff
@@ -0,0 +1,47 @@
+  selector:
+    matchLabels:
+      app: pg-avalanche
+  replicas: 1 # tells deployment to run 2 pods matching the template
```
The comment here is misleading.

csmarchbanks commented on kubernetes-deployment-yaml:
```diff
@@ -0,0 +1,47 @@
+    spec:
+      containers:
+      - name: pg-avalanche
+        image: quay.io/freshtracks.io/avalanche
```
Perhaps add a `:latest` tag to always pull down the latest image?

csmarchbanks commented on kubernetes-deployment-yaml:
```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: StatefulSet
```
Any reason this is a StatefulSet instead of a Deployment? If not I find Deployments easier to work with.

ravishankarsrrav commented:
Thank you for reviewing the contribution. I have modified the yaml file as changes suggested by you. It makes sense to keep it in a separate folder so I have created a new k8s folder and added the deployment yaml file.

csmarchbanks commented:
This looks great, thanks for your contribution!
